packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -15,42 +14,48 @@ const messages = ${JSON.stringify(messages, null, 2)};
 module.exports = {
   areTranslationsLoaded: true,
   locale: '${language}',
-  messages,
+  messages: messages
 };`;
 }
 
-function getDirectories(srcPath) {
-  return fs.readdirSync(srcPath).filter(file => fs.statSync(path.join(srcPath, file)).isDirectory());
+function getDirectories(srcPath, compiler) {
+  return compiler.inputFileSystem.readdirSync(srcPath).filter(file => compiler.inputFileSystem.statSync(path.join(srcPath, file)).isDirectory());
 }
 
-function aggregateDirectory(languageMessages, currentDirectory) {
+function aggregateDirectory(languageMessages, currentDirectory, compiler) {
   // Check the directory for translations
   const translationsDirectory = path.resolve(currentDirectory, 'translations');
-  if (fs.existsSync(translationsDirectory)) {
+  try {
+    // Check if the directory exists by attempting to read from it
+    compiler.inputFileSystem.readdirSync(translationsDirectory);
+
     // Check the directory for each translation file
     supportedLocales.forEach((language) => {
       const translationFile = path.resolve(translationsDirectory, `${language}.json`);
-      if (fs.existsSync(translationFile)) {
-        Object.assign(languageMessages[language], JSON.parse(fs.readFileSync(translationFile, 'utf8')));
-      } else {
-        // eslint-disable-next-line no-console
+      try {
+        Object.assign(languageMessages[language], JSON.parse(compiler.inputFileSystem.readFileSync(translationFile, 'utf8')));
+      } catch (e) {
         console.warn(`Translation file ${language}.json not found for ${translationsDirectory}`);
       }
     });
+  } catch (e) {
+    // eslint-disable-line no-use-before-defin
   }
 
   // Check the directory's node_modules for translation files
   const nodeMoudlesPath = path.resolve(currentDirectory, 'node_modules');
-  if (fs.existsSync(nodeMoudlesPath)) {
-    getDirectories(nodeMoudlesPath).forEach((module) => {
-      aggregateDirectory(languageMessages, path.resolve(nodeMoudlesPath, module));
+  try {
+    getDirectories(nodeMoudlesPath, compiler).forEach((module) => {
+      aggregateDirectory(languageMessages, path.resolve(nodeMoudlesPath, module), compiler);
     });
+  } catch (e) {
+    // eslint-disable-line no-use-before-defin
```

Comment:
Should we output information in these catch's?

Reply:
It was decided to ignore outputting anything here, I was outputting the error here, however that provided many more logs that previously were not logged out.  They were also causing the CI job to fail as the output became to large.  Before, we were using existsSynch and did nothing in the case when the directory didn't exist.  Now we try to read from it instead of using existsSynch to determine if the directory is there.